src/libraries/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -13,6 +13,8 @@
          TODO: [AndroidCrypto] Remove ANDROID_OPENSSL_AAR check once Android native crypto is complete -->
     <UseAndroidCrypto Condition="'$(TargetsAndroid)' == 'true' and '$(ANDROID_OPENSSL_AAR)' == ''">true</UseAndroidCrypto>
     <UseAppleCrypto Condition="'$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">true</UseAppleCrypto>
+    <IncludeAppleCryptoOnMobile Condition="'$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">false</IncludeAppleCryptoOnMobile>
+    <IncludeAppleCryptoOnMobile Condition="'$(IncludeAppleCryptoOnMobile)' == ''">true</IncludeAppleCryptoOnMobile>
```

Comment:
IncludeAppleCryptoOnMobile is true... except for on mobile?  That seems oddly named, to me.

Reply:
Renamed to `IncludeUnavailableAppleCryptoAPIs`